con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -3,21 +3,16 @@
 import org.hibernate.validator.constraints.Email;
 import org.hibernate.validator.constraints.NotBlank;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.validation.constraints.Max;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.Pattern;
-import javax.validation.constraints.Size;
+import javax.persistence.*;
```

Comment:
Remove wildcard imports as stated in [docs](https://app.nuclino.com/teams/13:30480/documents/General-pointers-4308315e-fc99-4f73-85dc-dd8a4d330620).